test/make-images-push-to-local-registry.sh

```diff
@@ -21,6 +21,12 @@ docker push "$1/cilium/operator-aws:$2"
 docker push "$1/cilium/operator-azure:$2"
 docker push "$1/cilium/hubble-relay:$2"
 
+# push startup-script image with proper tag to repo
+nodeInitTag="af2a99046eca96c0138551393b21a5c044c7fe79"
```

Comment:
This SHA may get out of sync over time? Would it be possible to pull it from `install/kubernetes/cilium/charts/nodeinit/values.yaml` instead:
```
grep "tag: " install/kubernetes/cilium/charts/nodeinit/values.yaml | cut -d ' ' -f2
```
Likely there is some nicer method to access that from Go, though?

Reply:
Let's merge this and fix this properly in follow-up PR.